src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,28 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        int count = 0;
+
+        if (a) {
+            count++;
+        }
+        
+        if (b) {
+            count++;
+        }
+        
+        if (c) {
+            count++;
+        }
+        
+        if (d) {
+            count++;
+        }
+
+        if (count == 2) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
this task can be solved in a single line

Reply:
Done